index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/dcd34e45ea.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+    <title>Global Summit</title>
+</head>
+<body>
+
+        <header>
+            <div class="upper-links">
+                <i class="fa-brands fa-facebook-f"></i>
+                <i class="fa-brands fa-twitter"></i>
+                <a href="#">Logout</a>
+                <a href="#">Mypage</a>
+                <a href="#">Korean</a>
```

Comment:
- [ ] For this list of options I suggest using an unordered list <ul> to meet the symantic html rules :recycle:

Reply:
I've converted them in unordered list on all places as instructed